modules/instruments/php/module.class.inc

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * This serves as a hint to LORIS that this module is a real module.
+ * It does nothing but implement the module class in the module's namespace.
+ *
+ * PHP Version 5
```

Comment:
```suggestion
  * PHP Version 7
```

Reply:
apply this as well

Reply:
I did it manually to avoid breaking PHPCS with the space at the end of the line.

Reply:
I care more about indicating the version rather than the spacing

Reply:
but travis cares about the spacing.